Userland/Libraries/LibGUI/EmojiInputDialog.cpp

```diff
@@ -184,8 +184,8 @@ auto EmojiInputDialog::supported_emoji() -> Vector<Emoji>
         auto button = EmojiButton::construct(move(code_points));
         button->set_fixed_size(button_size, button_size);
         button->set_button_style(Gfx::ButtonStyle::Coolbar);
-        button->on_click = [this, text](auto) {
-            m_selected_emoji_text = move(text);
+        button->on_click = [this, text = move(text)](auto) {
```

Comment:
This can't be moved here, it's used below, so this breaks some features of this dialog - please be sure to test this.

The way the capture is currently written `[this, text]` creates a copy of `text`. The `move` is marked redundant by clangd because this is missing a `mutable` specifier, thus `text` is `const`.
```c++
        button->on_click = [this, text](auto) mutable {
            m_selected_emoji_text = move(text);
            done(ExecResult::OK);
        };
```

Reply:
Ah okm did not see the other mention of `text`
But we cannot `move` in the lambda, because that would break when you press the button a second time

Reply:
There isn't a second time - the dialog is destroyed right after you click

Reply:
good to know, will change it appropriately